### PR TITLE
Refactored messageReaction for social media posting

### PR DESCRIPTION
**Pull request by ericm**
#10 
Tested and changes are confirmed working
Reduced verbosity a good bit

### CODE REVIEW

**Comment by ericm** [comment]
> where da heck did all the code disappear to???

Removed the duplicate code by using an interface ;)

**Comment by Strum355** [comment]
This says alot about ~~our society~~ the state of the old code :clown_face: 

does this actually *work*?

**Comment by ericm** [comment]
Yeah tested it on my dev server and all

File: commands/register.go [hunk 1]
```diff
@@ -149,18 +149,15 @@ func messageReaction(s *discordgo.Session, m *discordgo.MessageReactionAdd) {
...
-				fmt.Println(content.Description)
 				mediaIds := []int64{}
-				if content.Image != nil {
+				if content.GetImage() != nil {
 					// Contains image
 					// Upload image
-					getImage, err := http.Get(content.Image.Request.URL.String())
+					getImage, err := http.Get(content.GetImage().Request.URL.String())
 					if err != nil {
 						log.Error(err.Error())
```
Comment (Strum355) [inline commands/register.go:162]: `log.WithError(err).Error("failed to perform epic reddit gamer keanu reeves big chungus moment")`

File: commands/register.go [hunk 2]
```diff
@@ -173,18 +173,18 @@ func messageReaction(s *discordgo.Session, m *discordgo.MessageReactionAdd) {
 							log.Info(getImage.Header.Get("content-type"))
 							log.Info(mediaHTTP.Status)
 						} else {
```
Comment (Strum355) [inline commands/register.go:175]: Are these `else` necessary? Should the function not return if theres an error or what should happen that means we cant remove the else and indent the logging out by one?

**Comment by Thomas007G** [comment]
Looks good 👌

**Pull request merged**